### PR TITLE
Matter: remove device from fabric

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/109127
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/matter.markdown [hunk 1]
```diff
@@ -255,6 +255,19 @@ Follow these steps if you have added a Matter device to Home Assistant and you w
    - For example, if you shared it with Google Home, it lists **Google LLC**.
      ![image](/images/integrations/matter/matter_share-device-with-other-platform.png)
 
+## Removing a device from a network
+
+Follow these steps if you want to remove a device from a particular network.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
Follow these steps if you want to remove a device from a particular Matter controller.
```

File: source/_integrations/matter.markdown [hunk 2]
```diff
@@ -255,6 +255,19 @@ Follow these steps if you have added a Matter device to Home Assistant and you w
...
+
+1. Go to {% my integrations title="**Settings** > **Devices & services**" %} and on the **Matter (BETA)** integration card, select **Devices**.
+2. From the list of devices, select the device you want to remove from a network.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]: I do not like the term network here. Can we stick to Matter fabric or Matter controller. Or maybe even app ?

File: source/_integrations/matter.markdown [hunk 3]
```diff
@@ -255,6 +255,19 @@ Follow these steps if you have added a Matter device to Home Assistant and you w
...
+3. In the **Device info** section, next to **Share device**, select the three dot menu. Then, select **Manage fabrics**.
+4. From the list, remove the network of interest. 
+   - If you want to remove Apple Home, also remove the Apple Keychain entry.
+   ![image](/images/integrations/matter/matter-remove-from-network.png)
+
+
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown:296]: Another note could be that if you want to remove the device from the home Assistant fabric, you need to press remove from the 3 dots context menu

File: source/_integrations/matter.markdown (hunk 2 above)
Comment (c0ffeeca7) [reply source/_integrations/matter.markdown]: so, for me, it does not remove it from the app. It is still in the app, but unresponsive. It just removes it from the Home Assistant UI.

File: source/_integrations/matter.markdown (hunk 3 above)
Comment (c0ffeeca7) [reply source/_integrations/matter.markdown:296]: added a step

File: source/_integrations/matter.markdown [hunk 4]
```diff
@@ -255,6 +255,19 @@ Follow these steps if you have added a Matter device to Home Assistant and you w
...
+Follow these steps if you want to remove a device from a particular Matter controller.
+
+1. Go to {% my integrations title="**Settings** > **Devices & services**" %} and on the **Matter (BETA)** integration card, select **Devices**.
+2. From the list of devices, select the device you want to remove from a controller.
+3. In the **Device info** section, next to **Share device**, select the three-dot menu. Then, select **Manage fabrics**.
+4. From the list, remove the controller of interest.
+   - If you want to remove Apple Home, also remove the Apple Keychain entry.
+   ![image](/images/integrations/matter/matter-remove-from-network.png)
+5. If you want to remove the device from Home Assistant, select the three-dot menu and select **Delete**.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]:
```suggestion
5. If you want to remove the device from Home Assistant itself, select the three-dot menu and select **Delete**.
```

Comment (marcelveldt) [reply source/_integrations/matter.markdown]: just a suggestion to add the word itself to make it more explicit ?

**Review by marcelveldt — APPROVED** [review]
nice work again!

**Pull request merged**